### PR TITLE
Can display hello react

### PR DESCRIPTION
**Pull request by dhagandev**

### CODE REVIEW

File: src/js/react.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+function run() {
+    ReactDOM.render(
+        React.createElement(App),
+        document.getElementById("reactComponent")
+    );
+}
+
+const loadedStates = ['complete', 'loaded', 'interactive'];
+
+if (loadedStates.includes(document.readyState) && document.body) {
```
Comment (kev-chan) [inline src/js/react.js]: what is this entire if/else block used for?

File: src/index.html [hunk 2]
```diff
@@ -1,10 +1,14 @@
 <html>
     <head>
         <title>Lithium Onboarding</title>
+        <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+        <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+        <script src="js/react.js"></script>
```
Comment (kev-chan) [inline src/index.html]: get a better name, this sounds like it is react you are loading

File: src/js/react.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class App extends React.Component {
```
Comment (kev-chan) [inline src/js/react.js]: why class 'App'?

**Review by kev-chan — CHANGES_REQUESTED** [review]
same high level question i asked on Friday:
- why does this work as opposed to other solutions you tried?

**Comment by dhagandev** [comment]
Not sure how to respond directly to the "why does this work as opposed to other solutions you tried?" comment, there is no "reply" like the comments below. But this way works because I am including React and React Dom in the html files. The other ways I had been trying were to add React/React Dom to the js file rendering the React components. Import and require are not implemented in any client side Javascript natively. I hadn't considered adding them to the html originally because it doesn't feel like the best way to be doing this. All the tutorials suggest imports, but mention needing extra libraries like ES6, Babel, etc.

**Pull request merged**